### PR TITLE
Rebuild the charts for Meshery and the meshery-operator as the primary chart

### LINKED ISSUE #4520: Migration the Helm chart of Meshery to support v0.6.0 release
#### Current Behavior

Meshery-server as the main chart

#### Desired Behavior

Meshery-operator as the main chart

#### Implementation

A question may exsit here:
 
Do we allow meshery was deploy directly without meshery-operator?

* If it is ture, should we keep the meshery helm charts and sub-charts?

* If it isn't, we need to change the chart's name to meshery-operator and there is not exist a independen meshery charts

* How can we deal with the user update to the new version?

#### Mockups

---
#### Contributor [Guides](https://docs.meshery.io/project/contributing) and Resources
- 🛠 [Meshery Build & Release Strategy](https://docs.meshery.io/project/build-and-release)
- 📚 [Instructions for contributing to documentation](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#documentation-contribution-flow)
   - Meshery documentation [site](https://docs.meshery.io/) and [source](https://github.com/meshery/meshery/tree/master/docs)
- 🎨 Wireframes and designs for Meshery UI in [Figma](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI)
- 🙋🏾🙋🏼 Questions: [Layer5 Discussion Forum](https://discuss.layer5.io) and [Layer5 Community Slack](http://slack.layer5.io)

### PR DESCRIPTION
**Pull request by Aisuko**
Signed-off-by: aisuko <urakiny@gmail.com>

**Description**

As we disscused https://layer5io.slack.com/archives/C010LFFGFFA/p1634747627049700

This PR fixes #4520 

**Notes for Reviewers**

I test the several main components and looks like the here working well, but may exist other potential issues and need more test case.

```
➜  cluster kubectl get pods -A -w       
NAMESPACE       NAME                                        READY   STATUS      RESTARTS   AGE
ingress-nginx   nginx-ingress-controller-dtfbn              1/1     Running     0          7m22s
kube-system     calico-kube-controllers-6c977d77bc-ch7j6    1/1     Running     0          7m41s
kube-system     canal-dxb65                                 2/2     Running     0          7m40s
kube-system     coredns-685d6d555d-qj526                    1/1     Running     0          7m41s
kube-system     coredns-autoscaler-57fd5c9bd5-7rkb9         1/1     Running     0          7m41s
kube-system     metrics-server-7bf4b68b78-5gc8j             1/1     Running     0          7m42s
kube-system     rke-coredns-addon-deploy-job-f5psl          0/1     Completed   0          6h30m
kube-system     rke-ingress-controller-deploy-job-x8drl     0/1     Completed   0          6h29m
kube-system     rke-metrics-addon-deploy-job-272l9          0/1     Completed   0          6h29m
kube-system     rke-network-plugin-deploy-job-d6hmt         0/1     Completed   0          6h30m
meshery         meshery-97565c89f-7tjdb                     1/1     Running     0          13s
meshery         meshery-meshery-operator-7f86696d85-7r5qp   2/2     Running     0          13s                                                                                                                                                                                                                                 
➜  cluster kubectl logs -f meshery-meshery-operator-7f86696d85-7r5qp -c manager -n meshery 
time="2021-10-29T08:47:40Z" level=info msg="metrics server is starting to listen" app=meshery-operator
time="2021-10-29T08:47:40Z" level=info msg="reconciler groupmeshery.layer5.ioreconciler kindMeshSync" app=meshery-operator
time="2021-10-29T08:47:40Z" level=info msg="reconciler groupmeshery.layer5.ioreconciler kindBroker" app=meshery-operator
time="2021-10-29T08:47:40Z" level=info msg="starting manager" app=meshery-operator
time="2021-10-29T08:47:40Z" level=info msg="starting metrics server" app=meshery-operator
I1029 08:47:40.238623       1 leaderelection.go:248] attempting to acquire leader lease meshery/operator-12f37cee-dcc4-4ce8-923d-05f1720c5b40.meshery.layer5.io...
I1029 08:47:40.286314       1 leaderelection.go:258] successfully acquired lease meshery/operator-12f37cee-dcc4-4ce8-923d-05f1720c5b40.meshery.layer5.io
time="2021-10-29T08:47:40Z" level=info msg=Normal app=meshery-operator
time="2021-10-29T08:47:40Z" level=info msg=Normal app=meshery-operator
time="2021-10-29T08:47:40Z" level=info msg="Starting EventSource" app=meshery-operator
time="2021-10-29T08:47:40Z" level=info msg="Starting Controller" app=meshery-operator
time="2021-10-29T08:47:40Z" level=info msg="Starting EventSource" app=meshery-operator
time="2021-10-29T08:47:40Z" level=info msg="Starting Controller" app=meshery-operator
time="2021-10-29T08:47:40Z" level=info msg="Starting workers" app=meshery-operator
time="2021-10-29T08:47:40Z" level=info msg="Starting workers" app=meshery-operator
ç^C
➜  cluster kubectl logs -f meshery-97565c89f-7tjdb -n meshery                                                    
time="2021-10-29T08:47:39Z" level=info msg="Registered Meshery local Capabilities"
time="2021-10-29T08:47:39Z" level=info msg="Meshery server current channel: stable"
time="2021-10-29T08:47:39Z" level=info msg="Using '/home/appuser/.meshery/config' to store user data"
time="2021-10-29T08:47:39Z" level=info msg="Using '/home/appuser/.kube' as the folder to look for kubeconfig file"
time="2021-10-29T08:47:39Z" level=info msg="Log level: info"
time="2021-10-29T08:47:39Z" level=info msg="[SEEDING] Extracting Patterns from /home/appuser/.meshery/seed_content/patterns" app=meshery
time="2021-10-29T08:47:39Z" level=info msg="Starting to seed patterns" app=meshery
time="2021-10-29T08:47:39Z" level=info msg="[SEEDING] Saving pattern- IstioFilterPattern.yaml" app=meshery
time="2021-10-29T08:47:39Z" level=info msg="[SEEDING] Saving pattern- bookInfoPattern.yaml" app=meshery
time="2021-10-29T08:47:40Z" level=info msg="[SEEDING] Saving pattern- BookinfoDelayedVS.yaml" app=meshery
time="2021-10-29T08:47:40Z" level=info msg="[SEEDING] Saving pattern- BookinfoFaultedVS.yaml" app=meshery
time="2021-10-29T08:47:40Z" level=info msg="[SEEDING] Saving pattern- bookinfo.yaml" app=meshery
time="2021-10-29T08:47:40Z" level=info msg="[SEEDING] Saving pattern- bookinfovs.yaml" app=meshery
time="2021-10-29T08:47:40Z" level=info msg="[SEEDING] Saving pattern- bookinfowithcircuitbreaker.yaml" app=meshery
time="2021-10-29T08:47:40Z" level=info msg="[SEEDING] Saving pattern- bookinfowithretries.yaml" app=meshery
time="2021-10-29T08:47:40Z" level=info msg="[SEEDING] Saving pattern- circuitbreaker.yaml" app=meshery
time="2021-10-29T08:47:40Z" level=info msg="[SEEDING] Saving pattern- retries.yaml" app=meshery
time="2021-10-29T08:47:40Z" level=info msg="[SEEDING] Extracting Filters from /home/appuser/.meshery/seed_content/filters/binaries" app=meshery
time="2021-10-29T08:47:40Z" level=info msg="Starting to seed filters" app=meshery
time="2021-10-29T08:47:40Z" level=info msg="[SEEDING] Saving filter- http_auth_bg.wasm" app=meshery
time="2021-10-29T08:47:40Z" level=info msg="[SEEDING] Saving filter- jwt_filter_bg.wasm" app=meshery
time="2021-10-29T08:47:40Z" level=info msg="[SEEDING] Saving filter- metrics_collector_bg.wasm" app=meshery
time="2021-10-29T08:47:40Z" level=info msg="[SEEDING] Saving filter- singleton_http_call_bg.wasm" app=meshery
time="2021-10-29T08:47:40Z" level=info msg="[SEEDING] Saving filter- singleton_queue_bg.wasm" app=meshery
time="2021-10-29T08:47:40Z" level=info msg="[SEEDING] Saving filter- tcp_metrics_bg.wasm" app=meshery
time="2021-10-29T08:47:40Z" level=info msg="[SEEDING] Saving filter- tcp_packet_parse_bg.wasm" app=meshery
time="2021-10-29T08:47:40Z" level=info msg="[SEEDING] Extracting Applications from /home/appuser/.meshery/seed_content/applications" app=meshery
time="2021-10-29T08:47:40Z" level=error msg="lstat /home/appuser/.meshery/seed_content/applications: no such file or directory" app=meshery code=2153 probable-cause="Sample content does not exist.\nContent file format not supported.\nUser doesn't have permission to read sample content.\nContent file corrupt." severity=2 short-description="Error while getting .Applications. from sample content" suggested-remediation="Try restarting Meshery.\nTry fetching content again."
time="2021-10-29T08:47:41Z" level=info msg="Starting Server listening on :8080"
```


**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.
 

<!--
Thank you for contributing to Meshery! 

Contributing Conventions:

1. Include descriptive PR titles with [<component-name>] prepended.
2. Build and test your changes before submitting a PR. 
3. Sign your commits

By following the community's contribution conventions upfront, the review process will 
be accelerated and your PR merged more quickly.
-->

### CODE REVIEW

File: .github/workflows/helm-chart-releaser.yml [hunk 1]
```diff
@@ -29,8 +29,10 @@ jobs:
       with:
         version: 'v3.6.0' # default is latest stable
       id: install
-    - name: validate
+    - name: validate meshery
       run: helm lint install/kubernetes/helm/meshery --with-subcharts
+    - name: validate meshery-operator
```
Comment (Aisuko) [inline .github/workflows/helm-chart-releaser.yml:34]: Due to https://github.com/stefanprodan/helm-gh-pages/issues/23#issue-800058519, I'd like to make little change on the helm-charts CI.

File: install/kubernetes/helm/meshery-operator/values.yaml [hunk 2]
```diff
@@ -0,0 +1,176 @@
...
+testCase:
+  enabled: false
+
+
+# By default, the adapters need to have the same cluster role as the meshery to have permissions for performing necessary operations
+
+# If need to use a different service accounts for the adapters, modify the serviceAccountNameOverride for the corresponding adapters here,
+# or set the serviceAccountNameOverride to empty while setting the serviceAccount information in the sub-chart accordingly
+meshery-app-mesh:
+  enabled: false
```
[the remark this answers is not in the document]
Comment (navendu-pottekkat) [reply install/kubernetes/helm/meshery-operator/values.yaml:108]: We have documented this in Meshery Docs here: https://docs.meshery.io/installation/platforms/kubernetes#using-helm-charts

**Review by leecalcote — APPROVED** [review]
I think we're there. @panyuenlau any final reservations?

**Review by utkarsh-pro — APPROVED** [review]
LGTM :raised_hands:

**Review by panyuenlau — APPROVED** [review]
Just tested with `mesheryctl` and everything seems to be working fine (hope I didn't miss anything)

**Review by navendu-pottekkat — APPROVED** [review]

**Pull request merged**